lienflash/lienflash-fe
# Sn setup token

#### What’s this PR do?
Implements authentication with JSON Web Token on the front end:
- Adds header with token to all API calls
- Adds GET request for user object to enable login to persist when page is refreshed
- Adds token to local storage on log-in and removes token when user logs out
- Updates conditional rendering of Login component so it will only be rendered if there if no token
- Updates POST/PATCH tests to pass with the token
- Edited the API endpoints to 'users/ plural due to change on the back end
#### Where should the reviewer start?
App.js 
#### How should this be manually tested?
In browser, if you login then refresh page it should reload App without losing your data.
#### Any background context you want to provide?
I made a couple minor changes that we needed to fix i.e. editing the job statuses on JobCard to enable the data to display correctly, and deleting a duplicate Homepage.scss file as we had two of them.
#### What are the relevant tickets?
closes #80 
#### Screenshots (if appropriate)
#### Questions:
#### Need Help/Support
n/a
